catlin/go.mod

```diff
@@ -3,16 +3,13 @@ module github.com/tektoncd/plumbing/catlin
 go 1.14
 
 require (
+	github.com/google/go-containerregistry v0.1.1
 	github.com/spf13/cobra v1.0.0
 	github.com/tektoncd/pipeline v0.15.2
 	go.uber.org/zap v1.15.0
 	gotest.tools/v3 v3.0.2
-
```

Comment:
could you please revert back unrelated change?

Reply:
`go mod tidy` removed them I guess :thinking: